packages/main/test/pages/CheckBox.html

```diff
@@ -83,6 +94,11 @@
 				event.preventDefault();
 			});
 		});
-</script>
+
+		cbForm.addEventListener("submit", function(event) {
+			event.preventDefault();
+			alert("Form submitted!");
```

Comment:
Minor: maybe we can enhance this handler a little bit and to add test about this functionality.

Reply:
You can check formElement.checkValidity() (this is the method that does the native validation of the forms). You can also use it for the test